apps/etterlatte-behandling/src/main/kotlin/oppgave/OppgaveService.kt

```diff
@@ -167,15 +168,27 @@ class OppgaveService(
         oppgaveDao.oppdaterReferanseOgMerknad(oppgaveId, referanse, merknad)
     }
 
-    fun endrePaaVent(paavent: PaaVent): OppgaveIntern {
-        val oppgave = hentOppgave(paavent.oppgaveId)
-        if (paavent.paavent && oppgave.status == Status.PAA_VENT) return oppgave
-        if (!paavent.paavent && oppgave.status != Status.PAA_VENT) return oppgave
+    fun endrePaaVent(
+        oppgaveId: UUID,
+        paavent: Boolean,
+        merknad: String,
+        aarsak: PaaVentAarsak?,
+    ): OppgaveIntern {
+        val oppgave = hentOppgave(oppgaveId)
+        if (paavent && oppgave.status == Status.PAA_VENT) return oppgave
+        if (!paavent && oppgave.status != Status.PAA_VENT) return oppgave
+
+        if (paavent && aarsak == null) {
```

Comment:
Det er nå jeg lurer på om det er smudere å splitte den metoden her i to, en for å sette på vent og en for å ta av vent..

Reply:
Kanskje, men litt større endring enn bugfixmat tenker jeg